README.md

```diff
@@ -7,31 +7,53 @@ can then perform whatever actions it wants.
 
 ## Basic usage
 
-`snare` requires three command-line arguments to be specified:
+`snare` has the following command-line format:
 
 ```
-Usage: snare [-e email] [-j <max-jobs>] -p <port> -r <repos-dir> -s <secrets-path>
+Usage: snare [-c <config path>]
```

Comment:
`<config-path>` with a hyphen, like before?

Reply:
Fixed in https://github.com/softdevteam/snare/pull/12/commits/b32b12216bad9fbea17e2d48169eafe7f3a9475f.